### PR TITLE
Make keyword passing MoPub compatible

### PR DESCRIPTION
**Pull request by chauduyphanvu1**
Convert birthday and gender to MoPub-parsable data.

### CODE REVIEW

**Comment by chauduyphanvu1** [comment]
@rsatrasala Review reminder ;)

File: ThirdPartyAdapters/mopub/mopub/src/main/java/com/mopub/mobileads/dfp/adapters/MoPubAdapter.java [hunk 1]
```diff
@@ -188,8 +191,6 @@ public void onDownloadFailure() {
                                 }).execute(map);
                             } catch (Exception e) {
                                 Log.d(TAG, "Exception constructing the native ad");
-                                listener.onAdFailedToLoad(
```
Comment (rsatrasala) [inline ThirdPartyAdapters/mopub/mopub/src/main/java/com/mopub/mobileads/dfp/adapters/MoPubAdapter.java]: Why are you removing this?

File: ThirdPartyAdapters/mopub/mopub/src/main/java/com/mopub/mobileads/dfp/adapters/MoPubAdapter.java [hunk 2]
```diff
@@ -303,16 +303,40 @@ public View getBannerView() {
...
+        }
+
         StringBuilder keywordsBuilder = new StringBuilder();
 
         keywordsBuilder = keywordsBuilder.append(MOPUB_NATIVE_CEVENT_VERSION)
-                .append(mediationAdRequest.getBirthday() != null ?
-                        ",m_birthday:" + mediationAdRequest.getBirthday() : "")
-                .append(mediationAdRequest.getGender() != -1 ?
-                        ",m_gender:" + mediationAdRequest.getGender() : "");
+                .append(ageString)
```
Comment (rsatrasala) [inline ThirdPartyAdapters/mopub/mopub/src/main/java/com/mopub/mobileads/dfp/adapters/MoPubAdapter.java]: Append the delimiters here so the code is easily extendable

**Comment by rsatrasala** [comment]
Minor comments. Otherwise LGTM

**Pull request closed without merging**